demos/basics/loading-pandas.ipynb

```diff
@@ -0,0 +1,3961 @@
+{
```

Comment:
<p>maybe just "construction from these DataFrames" (i.e. remove "one of") since we actually need two dataframes?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1074/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>